CMakeLists.txt

```diff
@@ -206,7 +206,15 @@ if(VLLM_GPU_LANG STREQUAL "CUDA")
   # Set CUTLASS_REVISION manually -- its revision detection doesn't work in this case.
   set(CUTLASS_REVISION "v3.5.1" CACHE STRING "CUTLASS revision to use")
 
-  FetchContent_Declare(
+  # Use the specified CUTLASS source directory for compilation if CUTLASS_SRC_DIR is provided
+  if (DEFINED ENV{CUTLASS_SRC_DIR})
+    set(CUTLASS_SRC_DIR $ENV{CUTLASS_SRC_DIR})
+  endif()
+
+  if(CUTLASS_SRC_DIR)
```

Comment:
I think we should probably name the env variable `VLLM_CUTLASS_SRC_DIR` for namespacing